requirements.txt

```diff
@@ -3,7 +3,4 @@ mkdocs-material>=5.4
 pymdown-extensions>=7.0.0
 mkdocs-branchcustomization-plugin~=0.1.3
 mkdocs-macros-plugin
-numpy
-pandas
-PYYaml
-tabulate
```

Comment:
[process_markdowns.py](https://github.com/bids-standard/bids-specification/blob/master/pdf_build_src/process_markdowns.py) depends on numpy, and depends on it being in requirements.txt --> https://github.com/bids-standard/bids-specification/blob/89115cd9ef0d4676ae78cfa1e9e78fa1048869a2/.circleci/config.yml#L66

It's also mentioned in the docs: https://github.com/bids-standard/bids-specification/blob/master/pdf_build_src/README.md

I don't know why the job still seems to pass :shrug: 

Could you either:
1. remove the numpy dependency in process_markdowns.py (preferred, seems possible at first glance, maybe even easy)
2. update the documentation (pdf build README) and ensure that numpy is installed whenever someone wants to build the pdf?
3. any other solution you might have,

Reply:
It looks like the `requirements.txt` file includes `schemacode`, which, in turn, includes `numpy` as a dependency. That explains why it's working. However, I agree that `numpy` probably shouldn't have been removed from the main requirements file, since it's used for specification code outside of `schemacode`. I'll add it back in.

The other requirements that were removed from that file all seem to be `schemacode`-specific, so I'll keep them out.

Reply:
I added it back in in f766804.

Reply:
> It looks like the requirements.txt file includes schemacode, which, in turn, includes numpy as a dependency. That explains why it's working.

thanks!

> I'll add it back in.

:+1: 

> The other requirements that were removed from that file all seem to be schemacode-specific, so I'll keep them out.

excellent, thanks